mmv1/products/netapp/activeDirectory.yaml

```diff
@@ -130,6 +130,12 @@ properties:
       Domain user/group accounts to be added to the Backup Operators group of the SMB service. The Backup Operators group allows members to backup and restore files regardless of whether they have read or write access to the files. Comma-separated list.
     required: false
     item_type: Api::Type::String
+  - !ruby/object:Api::Type::Array
+    name: 'administrators'
```

Comment:
on first glance, i dont see this new field being added to any tests. Because of this MM will return a fail.

Reply:
Done added the field to resource tests.

Reply:
@BBBmau can you please provide approval on this. PR is awaiting approval.